### ISSUE TITLE
Start an Import from the Browse screen of the Import Configuration

### ISSUE THREAD

**Issue by mariodavid** [mariodavid/cuba-component-data-import#32]
```
As a user of the import system
When I open the import configuration browse screen
Then I want to be able to select an existing import configuration and trigger an import
Where the only parameter is the import file
```

**Comment by mariodavid** [comment]
I implemented a simple version of the solution in here: https://github.com/mariodavid/cuba-component-data-import/pull/45 (especially: https://github.com/mariodavid/cuba-component-data-import/pull/45/commits/78b1f09a7e44818bb1abf87665955127407ad816)

Unfortunately I don't really have a good plan on how to make the solution DRY, because there will be basically three wizards:
1. full wizard with: "upload file", "selecting entity", "creating entity configuration", "import preview / manual editing" (the already existing one)
2. wizard with pre-selected import configuration: "upload file", "import preview / manual editing" (the current issue)
3. wizard with selected entity but without import configuration: "selecting import configuration", "upload file",  "import preview / manual editing"  (https://github.com/mariodavid/cuba-component-data-import/issues/33)


A few options that came into my mind:
* make the tabs content in frames for easily share in different wizards
* create an abstract superclass for the different kinds of wizards
* create a single wizard with different modes / switches

Perhaps we can share some ideas how to create a proper abstraction with the [ImportWizard](https://github.com/mariodavid/cuba-component-data-import/blob/master/modules/web/src/de/diedavids/cuba/dataimport/web/importwizard/ImportWizard.groovy) @aleksey-stukalov @knstvk? 

That would be great. Thx

**Comment by knstvk** [comment]
I would go for the first option - making the tab content in frames. We do it for large screens very often, even when we do not reuse the frames, just for maintainability. Unfortunately, I cannot suggest any specific details without diving into the implementation code, which I cannot afford now :)

**Comment by mariodavid** [comment]
Thanks for the info. Based on this comment as well as further ideas on that topic I created this app component which will hopefully be the baseline of this wizard. Although i'm not really sure when this will be ready to be used :)

https://github.com/mariodavid/cuba-component-wizard

**Comment by mariodavid** [comment]
implemented mostly a copy in #76 . Once the wizard is properly implemented i'll do a proper refactoring.
